napari/_app_model/actions/_layer_actions.py

```diff
@@ -209,3 +209,38 @@
             menus=[{'id': MenuId.LAYERS_PROJECT}],
         )
     )
+
+
+# Layer submenus
+
+LAYER_SUBMENUS = [
```

Comment:
nit: could we put the submenus at the top of the file, since the `LAYER_ACTIONS` refer to the submenu?

Reply:
Done for all.